apps/web/app/environments/[environmentId]/surveys/SurveyList.tsx

```diff
@@ -85,6 +94,22 @@ export default function SurveysList({ environmentId }) {
     }
   };
 
+  const copyToOtherEnvironment= async (surveyId)=>{
```

Comment:
Please install and use the `prettier` VSCode plugin to format your changes into the right format (spacing, etc.).